src/Microsoft.DotNet.CodeFormatter.Analyzers/Microsoft.DotNet.CodeFormatter.Analyzers.csproj

```diff
@@ -77,7 +77,6 @@
     <Compile Include="OptimizeNamespaceImportsFixer.cs" />
     <Compile Include="OptimizeNamespaceImportsOptions.cs" />
     <Compile Include="OptionsHelper.cs" />
-    <Compile Include="ProvideExplicitVariableTypeOptions.cs" />
```

Comment:
These options files used to contain a single option only, 'Enabled'. Instead of requiring these be explicitly declared/maintained, the tool now generates an Enabled option for every check.